luigi/cmdline.py

```diff
@@ -40,5 +41,14 @@ def luigid(argv=sys.argv[1:]):
             logging.basicConfig(level=logging.INFO, format=luigi.process.get_log_format(),
                                 filename=os.path.join(opts.logdir, "luigi-server.log"))
         else:
-            logging.basicConfig(level=logging.INFO, format=luigi.process.get_log_format())
+            config = luigi.configuration.get_config()
+            logging_conf = None
+            if not config.getboolean('core', 'no_configure_logging', False):
+                logging_conf = config.get('core', 'logging_conf_file', None)
+                if logging_conf is not None and not os.path.exists(logging_conf):
+                    logging_conf = None
+            if logging_conf is not None:
```

Comment:
Eh.. Yea.

Actually I think the server should just plain out die and not start if the condition `logging_conf is not None and not os.path.exists(logging_conf)` holds true.